src/main/java/com/google/api/codegen/discovery/transformer/nodejs/NodeJSSampleMethodToViewTransformer.java

```diff
@@ -73,9 +73,16 @@ private SampleView createSampleView(SampleTransformerContext context) {
 
     List<SampleFieldView> fields = new ArrayList<>();
     for (FieldInfo field : methodInfo.fields().values()) {
+      String name = field.name();
+      // Since the requestBody is named `resource`, all fields named `resource`
+      // are renamed by the Node.js client library generator to `resource_`.
+      if (name.equals("resource")) {
+        name = "resource_";
```

Comment:
It is captured by baseline tests, but because this PR doesn't actually change the output it doesn't show up in the diff.